src/tribler-core/tribler_core/modules/metadata_store/restapi/tests/test_search_endpoint.py

```diff
@@ -52,6 +63,12 @@ async def test_search(enable_chant, enable_api, session):
     parsed = await do_request(session, 'search?txt_filter=needle&sort_by=name', expected_code=200)
     assert len(parsed["results"]) == 1
 
+    parsed = await do_request(session, 'search?txt_filter=needle&sort_by=name&max_rowid=20',  expected_code=200)
+    assert len(parsed["results"]) == 0
+
+    parsed = await do_request(session, 'search?txt_filter=needle&sort_by=name&max_rowid=200', expected_code=200)
+    assert len(parsed["results"]) == 1
+
```

Comment:
Extracting is fine in this case, but you'll have to convert the common part of the test into a `pytest` fixture to avoid copy-pasting it for the extracted test.

Reply:
Done